components/teams-service/README.md

```diff
@@ -29,7 +29,7 @@ make test_with_db
 If you wish to run the tests in VSCode or elsewhere, set the following in the proper environment:
 
 ```
-export PG_URL="postgresql://postgres@127.0.0.1:5432/teams_test?sslmode=disable&timezone=UTC"
+export PG_URL="postgresql://postgres@127.0.0.1:5432/teams_test?sslmode=disable&password=docker"
```

Comment:
The UTC was there for a reason, wasn't it? I believe some tests mysteriously broke without it?

Reply:
oops didn't mean the drop that